rust-toolchain.toml

```diff
@@ -3,7 +3,7 @@
 # https://github.com/rust-lang/rustup/issues/2686
 # The auto-installation behaviour in rustup will likely be removed:
 # https://github.com/rust-lang/rustup/issues/1397
-channel = "1.70.0"
+channel = "1.71.0"
```

Comment:
Polkadot is already using 1.73.0: https://github.com/paritytech/polkadot-sdk/releases/tag/polkadot-v1.3.0

Reply:
https://github.com/Snowfork/snowbridge/pull/1011/commits/f70142dcbd837e0f16a7df21ab06c7cea3f6f75c